runtime/manual/basics/testing/index.md

```diff
@@ -453,7 +453,7 @@ that provides test spies and can be used in Deno by importing it from a CDN,
 such as Skypack:
```

Comment:
Looks like the line above still references Skypack?

Reply:
@kwhinnery Thanks, I have updated the line to mention NPM instead.